### PR TITLE
importing license information for dcos 2.1

### PR DESCRIPTION
**Pull request by cmastr**
D2IQ-70381

Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.

D2IQ-70381 

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/mesosphere/dcos/2.1/release-notes/open-source-attribution/index.md [hunk 1]
```diff
@@ -9,184 +9,742 @@ model: /mesosphere/dcos/2.1/data.yml
...
+|alibaba/dragonwell8                                                                                                       |v8.2.2-GA                                |Unknown License                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```
Comment (kevinmiller-conley) [inline pages/mesosphere/dcos/2.1/release-notes/open-source-attribution/index.md:23]: <a href="https://spdx.org/licenses/GPL-2.0-with-classpath-exception.html">GNU General Public License v2.0 w/Classpath exception</a>

File: pages/mesosphere/dcos/2.1/release-notes/open-source-attribution/index.md [hunk 2]
```diff
@@ -9,184 +9,742 @@ model: /mesosphere/dcos/2.1/data.yml
...
+|AXX-AIAPI                                                                                                                 |1.1.0                                    |Unknown License                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```
Comment (kevinmiller-conley) [inline pages/mesosphere/dcos/2.1/release-notes/open-source-attribution/index.md:211]: <a href="https://spdx.org/licenses/GPL-3.0-or-later.html">GNU General Public License v3.0 or later</a>

File: pages/mesosphere/dcos/2.1/release-notes/open-source-attribution/index.md [hunk 3]
```diff
@@ -9,184 +9,742 @@ model: /mesosphere/dcos/2.1/data.yml
...
+|finagle-init                                                                                                              |18.7.0                                   |<a href="https://www.apache.org/licenses/LICENSE-2.0">Apache License, Version 2.0</a>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+|finagle-netty3                                                                                                            |18.7.0                                   |<a href="https://www.apache.org/licenses/LICENSE-2.0">Apache License, Version 2.0</a>
```
Comment (kevinmiller-conley) [inline pages/mesosphere/dcos/2.1/release-notes/open-source-attribution/index.md:620]: <a href="https://opensource.org/licenses/MIT">MIT License</a>

Comment (kevinmiller-conley) [inline pages/mesosphere/dcos/2.1/release-notes/open-source-attribution/index.md:621]: <a href="https://opensource.org/licenses/BSD-3-Clause">BSD 3-Clause License</a>

File: pages/mesosphere/dcos/2.1/release-notes/open-source-attribution/index.md [hunk 4]
```diff
@@ -9,184 +9,742 @@ model: /mesosphere/dcos/2.1/data.yml
...
+|finagle-init                                                                                                              |18.7.0                                   |<a href="https://www.apache.org/licenses/LICENSE-2.0">Apache License, Version 2.0</a>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+|finagle-netty3
```
Comment (pierrebeitz) [inline pages/mesosphere/dcos/2.1/release-notes/open-source-attribution/index.md:339]: a whole lot of the errors in this file are due to the usage of plain tags. if we used the markdown equivalent, things would look a tiny bit better.

```
# 🔴 
<a href="https://my.org">Some Content</a>

# ✅ 
[Some Content](https://my.org)
```

**Review by russurquhart1 — APPROVED** [review]

**Comment by cmastr** [comment]
Merging this as-is with plans to update it in the future.

**Pull request merged**